lib/registries/addon/components/side-nav/x-link/template.hbs

```diff
@@ -7,7 +7,7 @@
         ...attributes
     >
         <span local-class='Label'>
-            <FaIcon @icon={{@icon}} @fixedWidth={{true}} @class={{local-class 'Icon'}} />
+            <FaIcon @name={{@icon}} @fixedWidth={{true}} @class={{local-class 'Icon'}} />
```

Comment:
I'm pretty sure what you had was correct: https://github.com/martndemus/ember-font-awesome/blob/v3.1.1/addon/components/fa-icon.js#L51-L54

Reply:
whoops, I switched back and forth and dropped this one along the way. fixed!